distro/configs/odoo/initializer_config/models/ir.model.1000.csv

```diff
@@ -0,0 +1,95 @@
+"","sale.field_sale_order__fiscal_position_id",""
+"","sale_purchase_stock.field_sale_order__id",""
+"","sale_stock.field_sale_order__incoterm",""
+"","sale.field_sale_order__invoice_count",""
+"","sale.field_sale_order__invoice_ids",""
+"","sale.field_sale_order__invoice_status",""
+"","sale.field_sale_order__is_expired",""
+"","sale_stock.field_sale_order__json_popover",""
+"","sale.field_sale_order__medium_id",""
+"","sale.field_sale_order__message_attachment_count",""
+"","sale.field_sale_order__message_channel_ids",""
+"","sale.field_sale_order__message_follower_ids",""
+"","sale.field_sale_order__message_has_error",""
+"","sale.field_sale_order__message_has_error_counter",""
+"","sale.field_sale_order__message_has_sms_error",""
+"","sale.field_sale_order__message_ids",""
+"","sale.field_sale_order__message_is_follower",""
+"","sale.field_sale_order__message_main_attachment_id",""
+"","sale.field_sale_order__message_needaction",""
+"","sale.field_sale_order__message_needaction_counter",""
+"","sale.field_sale_order__message_partner_ids",""
+"","sale.field_sale_order__message_unread",""
+"","sale.field_sale_order__message_unread_counter",""
+"","sale_mrp.field_sale_order__mrp_production_count",""
+"","sale.field_sale_order__my_activity_date_deadline",""
+"","sale.field_sale_order__name",""
+"","sale.field_sale_order__note",""
+"","sale.field_sale_order__order_line",""
+"","sale.field_sale_order__origin",""
+"","sale.field_sale_order__partner_id",""
+"","sale.field_sale_order__partner_invoice_id",""
+"","sale.field_sale_order__partner_shipping_id",""
+"","sale.field_sale_order__payment_term_id",""
+"","sale_stock.field_sale_order__picking_ids",""
+"","sale_stock.field_sale_order__picking_policy",""
+"","sale.field_sale_order__pricelist_id",""
+"","sale_stock.field_sale_order__procurement_group_id",""
+"","sale_purchase.field_sale_order__purchase_order_count",""
+"","sale.field_sale_order__reference",""
+"","sale.field_sale_order__require_payment",""
+"","sale.field_sale_order__require_signature",""
+"","sale_management.field_sale_order__sale_order_option_ids",""
+"","sale_management.field_sale_order__sale_order_template_id",""
+"","sale_stock.field_sale_order__show_json_popover",""
+"","sale.field_sale_order__show_update_pricelist",""
+"","sale.field_sale_order__signature",""
+"","sale.field_sale_order__signed_by",""
+"","sale.field_sale_order__signed_on",""
+"","sale.field_sale_order__source_id",""
+"","sale.field_sale_order__state",""
+"","sale.field_sale_order__tag_ids",""
+"","sale.field_sale_order__team_id",""
+"","sale.field_sale_order__transaction_ids",""
+"","sale.field_sale_order__type_name",""
+"","sale.field_sale_order__user_id",""
+"","sale.field_sale_order__validity_date",""
+"","sale_stock.field_sale_order__warehouse_id",""
+"","sale.field_sale_order__website_message_ids",""
+"","sale.field_sale_order__write_date",""
+"","sale.field_sale_order__write_uid",""
+"","init.c018bfa6-6925-4166-9f84-eda9f63b7e6b",""

```

Comment:
This last row is newly introduced. Why do we need to re-add the above rows? My assumption is that they already exist. How about adding the last row only? What's the behavior?
**Disclaimer**: _This is purely thinking out loud. I don't know how the initializer works._

Reply:
Unfortunately, the importer will throw an error if the preceding lines are missing. It could be possible that this got addressed in later versions of Odoo.